combinations/hash.tsv

```diff
@@ -366,3 +366,4 @@ snp-pileup=0.6.2,r-facets=0.6.2,samtools=1.15.1,bcftools=1.15.1,perl=5.32.1
 biopython=1.79,coreutils=9.1,dna_features_viewer=3.1.2,gawk=5.1.0,matplotlib=3.6.2,pandas=1.5.2
 coreutils=9.1,gawk=5.1.0,grep=3.4,matplotlib=3.6.2,numpy=1.23.5,pandas=1.5.2,samtools=1.16.1
 gffutils=0.11.1,biopython=1.80,pyvcf=0.6.8
+python=3.10,biopython,openpyxl,pandas,rich,typer,xlsxwriter,numpy,polars,pyarrow
```

Comment:
It better to keep the version specific and remove python.

Try locally to run `conda create -n foo biopython openpyxl pandas rich typer xlsxwriter numpy polars pyarrow --dry-run` and then see which versions do fit.

Reply:
@bgruening Is it necessary to remove Python? we just need to keep track python version as well?

Reply:
Not necessary, but also not needed. You can keep it if you like.